multi-int-store/src/main/java/com/connexta/multiintstore/controllers/QueryController.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2019 Connexta, LLC
+ *
+ * Released under the GNU Lesser General Public License version 3; see
+ * https://www.gnu.org/licenses/lgpl-3.0.html
+ */
+package com.connexta.multiintstore.controllers;
+
+import com.connexta.multiintstore.models.SearchDocument;
+import com.connexta.multiintstore.services.api.SearchService;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(value = "/search")
+public class QueryController {
+
+  @Autowired private SearchService searchService;
```

Comment:
Move this into an autowired constructor 
```suggestion
private SearchService searchService;

@Autowired
public QueryController (SearchService searchService) {
    this.searchService = searchService;
}
```